### DISCUSSION TITLE
Missing Shared Environment Variables on project creation page

### DISCUSSION THREAD

**Discussion by thihathit** [vercel/community#4818] [Idea]
Recently i've requested a feature for [SEV](https://github.com/orgs/vercel/discussions/1008#discussioncomment-4098869) and you guys added it. Really appreciated.

However I've been experiencing a strange behavior, on project creation page, linking SEV isn't available.

<img width="186" alt="image" src="https://github.com/vercel/community/assets/30023628/c6047ffc-7fa9-4b51-89c9-31bb6615a1a9">

<img width="724" alt="image" src="https://github.com/vercel/community/assets/30023628/b7e1f6b9-cb9a-4870-b2e2-e275798e9705">

This causes failure on the first build due to missing env. So I had to go inside the project setting, link the env and then redeploy it. It'd be really cool and save more time & builds if SEV is supported on project creation page.

Thank you.

**Comment by thihathit** [comment]
@amyegan

**Reply by amyegan** [reply]
Thanks for flagging this! I shared your idea with the team
